game-core/src/test/java/games/strategy/triplea/xml/TestMapGameData.java

```diff
@@ -31,18 +31,24 @@
 
   GLOBAL1940("ww2_g40_balanced.xml"),
 
-  TEST("Test.xml"),
+  TEST("Test.xml", true),
 
   DELEGATE_TEST("DelegateTest.xml"),
 
-  GAME_EXAMPLE("GameExample.xml"),
+  GAME_EXAMPLE("GameExample.xml", true),
 
   TWW("Total_World_War_Dec1941.xml");
 
   private final String fileName;
+  private final boolean test;
 
-  TestMapGameData(final String value) {
+  TestMapGameData(final String value, final boolean test) {
```

Comment:
Not sure if it would be better to name this parameter consistently with whatever parameter name you choose in `GameParser`.